unit/doc.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
What should do about the copyright header?
`Copyright 2015 CoreOS, Inc.` doesn't seem to make sense anymore.

Or just drop it entirely from the files? I don't think it is required, at least our other projects don't include it every file either.

Reply:
Yes, as far as I know there's no requirement to have a copyright in every source file. Will remove.